File: src/main/java/com/curseclient/mixin/accessor/entity/AccessorEntityLivingBase.java
package com.curseclient.mixin.accessor.entity;

import net.minecraft.entity.EntityLivingBase;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

@Mixin(EntityLivingBase.class)
public interface AccessorEntityLivingBase {

    @Accessor("jumpTicks")
    int getJumpTicks();

    @Accessor("jumpTicks")
    void setJumpTicks(int value);
}

File: src/main/kotlin/com/curseclient/mixin/accessor/entity/AccessorEntityLivingBase.kt
package com.curseclient.mixin.accessor.entity

import net.minecraft.entity.EntityLivingBase
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

@Mixin(EntityLivingBase::class)
interface AccessorEntityLivingBase {
    @get:Accessor("jumpTicks")
    @set:Accessor("jumpTicks")
    var jumpTicks : Int
}